### DISCUSSION TITLE
Filename

### DISCUSSION THREAD

**Discussion by alexidad** [hyunbinseo/zoom-rec-dl#4] [Q&A] [answered]
Hi there! 

Based on the settings, I understand that the filename would look something like this:

**Hyubin's Personal Meeting Room Jan 9, 2023 0428PM**

Am I correct?

**Comment by hyunbinseo** [comment, marked as answer]
Filename provided by Zoom.
`GMT20221221-095959_Recording_as_1920x1040.mp4`

Default filename generated by script.
`Hyubin's Personal Meeting Room GMT20221221-095959_Recording_as_1920x1040.mp4`

---

(meeting topic if exsits) (original mp4 filename) (optional timestamp that starts with `@`)

https://github.com/hyunbinseo/zoom-rec-dl/blob/9d09c42a2058247650337897ffdbb3a9c47db6c1/index.mjs#L158-L168
